docs/changelog.rst

```diff
@@ -8,6 +8,9 @@ KSQL 5.2 includes new features, including:
 
 * Added a new family of UDFs for improved handling of URIs (e.g. extracting information/decoding information), see :ref:`UDF table <functions>` for all URL functions
 * Added ``LIMIT`` keyword support for ``PRINT`` (`#1316 <https://github.com/confluentinc/ksql/issues/1316>`_)
+* Added support for read-after-write consistency: new commands will not execute until previous commands have finished executing.
+  This feature is enabled by default in the CLI (`#2280 <https://github.com/confluentinc/ksql/pull/2280>`_)
+  and can be implemented by the user for the REST API (`#2159 <https://github.com/confluentinc/ksql/pull/2159>`_).
```

Comment:
I don't love this link since it's not a great reference for users in terms of explaining how to configure read-after-write consistency for the REST API, but I couldn't think of a better link. Perhaps I should update the PR description to include an example? Or I should add a section to the docs somewhere?

Reply:
I think if would be a good addition to the API docs: https://github.com/confluentinc/ksql/blob/master/docs/developer-guide/api.rst

Reply:
+1 to @rodesai's suggestion!

Reply:
Gave it a go. What do you think?